src/pages/instances/InstanceList.tsx

```diff
@@ -178,10 +184,14 @@ const InstanceList: FC = () => {
       return false;
     }
     if (
```

Comment:
why don't we do something like 

```
const matchedProfiles =  filters.profiles.length > 0 && filters.profiles.every((profile) => item.profiles.includes(profile));
const matchedClusterMembers = filters.clusterMembers.length > 0 && filters.clusterMembers.includes(item.location);
...

return matchedProfiles || matchedClusterMembers || ...
```

Reply:
I think early access is a bit simpler to understand here and also a bit quicker. wdyt?

Reply:
yeah true, we won't have to do a filter for each condition then, please disregard